template/client/src/index.tsx

```diff
@@ -6,14 +6,26 @@ import App from './App';
 import { createClient } from "./graphql/client";
 import './index.css';
 import * as serviceWorker from './serviceWorker';
+import { KeycloakProvider } from 'react-keycloak';
+
+import { init, AeroGearApp } from '@aerogear/app';
+import { Auth } from '@aerogear/auth';
+
+import mobileConfig from './mobile-services.json'
+import { AeroGearConfiguration, ConfigurationService } from "@aerogear/core";
+
+const app = init(mobileConfig as unknown as AeroGearConfiguration)
```

Comment:
for tutorial purposes I would put auth into separate file. 
I would also copy auth detection from showcase so we do not need to talk about at the beggining of the tutorial